packages/cli/src/commands/run.js

```diff
@@ -0,0 +1,55 @@
+import fs from 'fs'
+import path from 'path'
+
+import execa from 'execa'
+import terminalLink from 'terminal-link'
+
+import { getPaths } from 'src/lib'
+import c from 'src/lib/colors'
+
+export const command = 'run side'
```

Comment:
I'm sure this is a historical preference on my part, but I always expect this time of command to be `serve` or `start`. Just couldn't make sense of `run`, but I might be weird and/or in the minority. Easily defer to group vote or precident.

Reply:
I'm totally not opinionated on this @thedavidprice. Let's discuss it at the core meeting

Reply:
I like `start`